swag/blocks/swag_blocks.py

```diff
@@ -164,7 +167,7 @@ def execute(self, db: SwagChain):
         user_account.pending_style = Style(0)
 
 
-@attrs(frozen=True, kw_only=True)
+@attrs(frozen=True, kw_only=True, eq=False)
```

Comment:
?

Reply:
From what I understood, attrs got some trouble to generate the hash function for this class by using its attributes.

More info here : https://www.attrs.org/en/stable/hashing.html